packages/core-components/src/layout/HeaderTabs/HeaderTabs.tsx

```diff
@@ -109,13 +112,15 @@ export function HeaderTabs(props: HeaderTabsProps) {
       >
         {tabs.map((tab, index) => (
           <TabUI
-            {...tab.tabProps}
             data-testid={`header-tab-${index}`}
             label={tab.label}
             key={tab.id}
+            component={Link}
+            to={removeLeadingSlash(tab.id)}
```

Comment:
are you sure the `removeLeadingSlash` is required here & nothing that can be fixed by the `relative`  prop of the React Router `Link`: https://reactrouter.com/en/main/components/link#relative

Reply:
A relative <Link to> value (that does not begin with /) resolves relative to the parent route, which means that it builds upon the URL path that was matched by the route that rendered that <Link>.
for ex tab.id = '/ci-cd'  just removing slash

Reply:
yes. that's why `relative="path"` might work here instead. anyways - I approved & happy to go with whatever:)